src/api/entities/Participant.ts

```diff
@@ -40,12 +40,16 @@ export class Participant extends BaseModel {
   id!: number;
   name!: string;
   status!: ParticipantStatus;
+  allowSharing!: boolean;
+  location?: string;
 }
 
 export const ParticipantSchema = z.object({
-  id: z.number().optional(),
+  id: z.number(),
```

Comment:
Making this required will break create participant. You've got to be cautious re-using these.

Reply:
Oh, I didn't realise that! Thanks, I add a new partial schema for creation